mobile/native/src/ln_dlc/mod.rs

```diff
@@ -168,6 +169,13 @@ pub fn get_seed_phrase() -> Vec<String> {
     state::get_seed().get_seed_phrase()
 }
 
+pub fn get_maintenance_margin() -> Decimal {
+    match state::try_get_tentenone_config() {
+        Some(config) => Decimal::try_from(config.maintenance_margin).expect("to fit into decimal"),
+        None => dec!(0.1),
```

Comment:
Maybe good to log an error if this ever returns `None` or maybe even add an `expect` as it should always be set?
Otherwise this will just hide errors which are hard to find.